File: Javascript/1042.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ').map(n => Number(n));
let normal = [...lines]
let menor = lines.sort(function(a,b){return a-b})
console.log(`${menor[0]}\n${menor[1]}\n${menor[2]}\n`)
console.log(`${normal[0]}\n${normal[1]}\n${normal[2]}`)

File: beecrowd/python/1042.py
variavel = input().split()

A = int(variavel[0])
B = int(variavel[1])
C = int(variavel[2])

if A<B and A<C:
 if B<C:
    print(A)
    print(B)
    print(C)
 elif C<B:
    print(A)
    print(C)
    print(B)

if B<A and B<C:
 if A<C:
    print(B)
    print(A)
    print(C)
 elif C<A:
    print(B)
    print(C)
    print(A)
      

if C<A and C<B:
 if A<B:
    print(C)
    print(A)
    print(B)
 elif B<A:
    print(C)
    print(B)
    print(A)

print("")
print(A)
print(B)
print(C)
  